roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1774,6 +1782,11 @@ def init_local_facts(self, facts=None,
                     if isinstance(val, basestring):
                         val = [x.strip() for x in val.split(',')]
                     new_local_facts['docker'][key] = list(set(val) - set(['']))
+            # Convert legacy log_options comma sep string to a list if present:
+            if 'log_options' in new_local_facts['docker'] and \
+                    isinstance(new_local_facts['docker']['log_options'], basestring) and \
+                    new_local_facts['docker']['log_options'] != '':
```

Comment:
It's happening because in this case it's a split on an empty string, which ends up returning [""]. Indeed those empty strings do get cleared out right below, I can push the entire "docker" block above down below the cleanup loop, or I can make a new "docker" block below just for this, or we keep it as is with an extra empty string check.